app/views/api/v1/greetings/random.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Api::V1::Greetings#random</h1>
+<p>Find me in app/views/api/v1/greetings/random.html.erb</p>
```

Comment:
- [Optional] I think we don't required a view file for our api. The client side(React) is responsible for this :+1:

Reply:
Fixed.